Modules/CodeTrek/Resources/views/layouts/master.blade.php

```diff
@@ -1,7 +1,9 @@
 @extends('layouts.app')
 @section('js_scripts')
+<script src="{{ mix('js/codetrek.js') }}"></script>
 @endsection
 
 @section('css_scripts')
+<link href="{{ mix('/css/codetrek.css') }}" rel="stylesheet">
```

Comment:
I think it should be like this as js path is also provider like this, please check can confirm if this also works or not?
```suggestion
<link href="{{ mix('css/codetrek.css') }}" rel="stylesheet">
```

Reply:
yes, it is working.